swf/src/types/color_transform.rs

```diff
@@ -1,39 +1,24 @@
+use crate::{Color, Fixed8};
 use std::ops::{Mul, MulAssign};
-use swf::{Color, Fixed8};
 
-#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+#[derive(Clone, Debug, Eq, PartialEq)]
```

Comment:
I'd prefer to keep `Copy`. `ColorTransform` is only 16 bytes, which is the size of a fat pointer.